web/crux/src/websockets/dyo.ws.adapter.ts

```
@@ -124,25 +149,6 @@ export default class DyoWsAdapter extends AbstractWsAdapter {
     return super.close(server)
   }
 
-  private async authorize(client: WsClient): Promise<boolean> {
-    const ctx = new ExecutionContextHost([client.connectionRequest], null, this.authorize)
-    let authorized = false
-    try {
-      authorized = await this.authGuard.canActivate(ctx)
-    } catch {
-      /* empty */
```

Thanks for deleted the empty catch.